app/index.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import styled, { createGlobalStyle } from 'styled-components';
+
+import Canvas from './components/visualizer/canvas';
+import SidePanel from './components/visualizer/sidePanel';
+
+const GlobalStyle = createGlobalStyle`
+    html, body, #content {
+        height: 100%;
+        margin: 0;
+    }
+`;
+const StyledContent = styled.div`
+    align-items: center;
+    display: grid;
+    grid-template-columns: 30% 1fr 1fr;
+    grid-template-rows: 35% 1fr 1fr;
+    grid-template-areas: 
+            "upper vis vis"
+            "lower vis vis"
+            "lower vis vis";
+    height: 100%;
+    text-align: center;
+`;
+
+class IndexPage extends React.Component {
```

Comment:
We can also use a functional component here with a useState hook

Reply:
Now that's a cool idea :p